CONTRIBUTING.md

```diff
@@ -1,19 +1,35 @@
 # Contributing to Prisma Documentation
 
-We welcome all contributions to the Prisma docs! If you come across some documentation that you feel is missing or incorrect, please open an issue or submit a pull request.
+Thank you for looking to contribute to our docs! If you come across some documentation on [prisma.io/docs](https://www.prisma.io/docs) that you feel is missing or incorrect, please open an issue or submit a pull request.
```

Comment:
Thank you for looking to <- feels weird to me
Thank you for wanting to  <- feels better

Reply:
```suggestion
Thank you for wanting to contribute to our docs! If you come across some documentation on [prisma.io/docs](https://www.prisma.io/docs) that you feel is missing or incorrect, please open an issue or submit a pull request.
```